fcli-core/fcli-fod/src/main/java/com/fortify/cli/fod/report/helper/FoDReportStatus.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright 2021, 2023 Open Text.
+ *
+ * The only warranties for products and services of Open Text
+ * and its affiliates and licensors ("Open Text") are as may
+ * be set forth in the express warranty statements accompanying
+ * such products and services. Nothing herein should be construed
+ * as constituting an additional warranty. Open Text shall not be
+ * liable for technical or editorial errors or omissions contained
+ * herein. The information contained herein is subject to change
+ * without notice.
+ *******************************************************************************/
+package com.fortify.cli.fod.report.helper;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import java.util.ArrayList;
+import java.util.stream.Stream;
+
+public enum FoDReportStatus {
+    Started, Completed, Failed, Queued;
+
+    public int getReportStatusType() {
```

Comment:
Do we really need to translate between status number and name (i.e., do we need to send status number to FoD, or receive status number from FoD that we need to convert to status name)? If FoD also accepts/returns status names, then better to use those. Otherwise, instead of depending on enum ordinal, maybe better to map explicitly using an int field/constructor with enum entries like `Started(0)`, `Completed(1)`, ...

Reply:
Changed to int field/constructor with enum entries as above.